patternfly/patternfly-elements
# feat: Move Hugo to npm dependency

## Hugo dependency moved to npm

Installing Hugo locally has caused some challenges in the past.  This PR seeks to remedy that by using the hugo npm wrapper as a means of installing and running the commands.


### Related issue

- (#100) Description


### What has changed and why

This change updates: 
- dependencies
- npm scripts
- removes bash scripts

### Testing instructions

1. Pull down this branch and `npm i`
  a. If you previously had Hugo installed, run `brew uninstall hugo` to ensure you have a fresh machine.
2. To test the build, run `npm run docs:build`.  Validate that the files are built to: docs/public
3. Validate the server: `npm run docs:serve`.  Validate that the files are built and this URL is opened: [http://localhost:9002/patternfly-elements/](http://localhost:9002/patternfly-elements/)

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
